gh_test.go

```diff
@@ -85,25 +86,26 @@ func TestGQLClient(t *testing.T) {
 	os.Setenv("GH_CONFIG_DIR", tempDir)
 	os.Setenv("GH_TOKEN", "GH_TOKEN")
 
-	http := httpmock.NewRegistry(t)
-	http.Register(
-		httpmock.GQL("QUERY"),
-		httpmock.StringResponse(`{"data":{"viewer":{"login":"hubot"}}}`),
-	)
+	gock.New("https://api.github.com").
+		Post("/graphql").
+		MatchHeader("Authorization", "token GH_TOKEN").
+		BodyString("QUERY").
+		Reply(200).
+		JSON(`{"data":{"viewer":{"login":"hubot"}}}`)
 
-	client, err := GQLClient(&api.ClientOptions{Transport: http})
+	client, err := GQLClient(nil)
 	assert.NoError(t, err)
 
 	vars := map[string]interface{}{"var": "test"}
 	res := struct{ Viewer struct{ Login string } }{}
 	err = client.Do("QUERY", vars, &res)
 	assert.NoError(t, err)
+	assert.True(t, gock.IsDone())
```

Comment:
If this is false, the resulting test failure won't be very descriptive. Is there a way to output the information about stubs that weren't matched?